src/app/shared/components/auth-service/auth.service.ts

```diff
@@ -0,0 +1,33 @@
+import { UserInfo } from '../../models/userInfo.model';
+
+export class AuthService {
+  public isLoggesIn: boolean = false;
+
+  constructor() {
+    this.checkIsLoggedIn();
+  }
+
+  private checkIsLoggedIn(): void {
+    const token: string = localStorage.getItem('userData');
+    if (token) {
+      this.isLoggesIn = true;
+    }
+  }
+
+  public logIn(): void {
+    localStorage.setItem('userData', JSON.stringify({ login: 'myLogin', password: 'myPassword', token: 'myToken' }));
+
```

Comment:
the `isLoggesIn ` property might need to be updated on login and logout actions

Reply:
done